frontend/src/app/supportAdmin/PendingOrganizations/PendingOrganizations.tsx

```diff
@@ -268,7 +272,7 @@ const PendingOrganizations = ({
                     <th scope="row">Contact information</th>
                     <th scope="row">Created</th>
                     <th scope="col">External ID</th>
-                    <th scope="col"></th>
+                    <th scope="col" aria-hidden></th>
```

Comment:
What does this empty column do? I removed it and it looks the same still and both pass the axe scans. 
With column: 
![image](https://user-images.githubusercontent.com/10108172/203359233-59cec5ff-f4ed-47e1-aa7d-07db0522cae8.png)Without: 
![image](https://user-images.githubusercontent.com/10108172/203359144-c9327720-dd3d-450b-a881-9a04cee8be9a.png)

Reply:
So in the [table body the edit & trash button are each their own element in the row](https://github.com/CDCgov/prime-simplereport/blob/b34e52219e1b1a86612dc620b96aa306a7930d3c/frontend/src/app/supportAdmin/PendingOrganizations/PendingOrganizations.tsx#L209-L230), should we add another empty column in that case or is just 1 for both of them ok?

Reply:
I added another heading so the number matches the number of columns in the body. Thanks for catching this!